console.rb

```diff
@@ -1,10 +1,15 @@
+require "bundler/setup"
+require "pg"
+require "active_record"
+require "pry"
+
 require_relative "db/connection"
-require_relative "models/apartment"
-require_relative "models/tenant"
+require_relative "models/apartments"
+require_relative "models/tenants"
 
```

Comment:
It's possible the json error resulted from pluralizing the model-- the convention is that the model name is singular ie `apartment.rb` and `tenant.rb`